nautobot/core/testing/views.py

```diff
@@ -817,7 +825,7 @@ def test_queryset_to_csv(self):
             instance1_csv_data += instance1_cf_values
             # Since values in `data` are all in str; cast all values in instance1_csv_data to str
             instance1_csv_data = [str(val) for val in instance1_csv_data]
-            instance1_cf_headers = ["cf_" + str(cf.slug) for cf in instance1.get_custom_fields().keys()]
+            instance1_cf_headers = ["cf_" + str(cf.key) for cf in instance1.get_custom_fields().keys()]
```

Comment:
We really ought to have a standard helper function `cf -> "cf_{cf.key}"` and use it all over the place... - maybe an instance method or property on the CustomField class?